fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1251,6 +1255,7 @@ public OnlineIndexer build() {
         protected void validate() {
             validateIndex();
             validateLimits();
+            validateSubspaceProvider();
```

Comment:
If you do what I suggest above, then there is no reason for this validation.

Reply:
IndexBuilder requires a SubspaceProvider in various places in the code. Seemed like a good idea to validate that it had one.